Backend/app/rider-platform/rider-app/Main/src/Storage/Tabular/Ride.hs

```diff
@@ -72,8 +73,12 @@ instance TEntityKey RideT where
   fromKey (RideTKey _id) = Id _id
   toKey (Id id) = RideTKey id
 
-instance FromTType RideT Domain.Ride where
-  fromTType RideT {..} = do
+type FullRideT = (RideT, Loc.LocationT, [Loc.LocationT])
+
+instance FromTType FullRideT Domain.Ride where
+  fromTType (RideT {..}, fromLocT, mbToLocT) = do
+    fromLocation <- fromTType fromLocT
+    toLocation <- mapM fromTType mbToLocT
     tUrl <- parseBaseUrl `mapM` trackingUrl
```

Comment:
why from and to location is not added in tables  when it is there in domain.types